tests/utils.go

```diff
@@ -1441,14 +1436,19 @@ func RunOcCommand(args ...string) (string, error) {
 		args = append(args, "--server", master.String())
 	}
 
-	cmd := exec.Command(KubeVirtOcPath, args...)
+	cmd := exec.Command(cmdPath, args...)
 	kubeconfEnv := fmt.Sprintf("KUBECONFIG=%s", kubeconfig.String())
 	cmd.Env = append(os.Environ(), kubeconfEnv)
 
-	stdOutErrBytes, err := cmd.CombinedOutput()
+	switch cmdName {
```

Comment:
only change is here, seems `oc` and `kubectl` have to use different cmd Ouput to deal with errors.